versions.tf

```diff
@@ -5,7 +5,6 @@ terraform {
     aws        = ">= 3.35.0"
     local      = ">= 1.4"
     null       = ">= 2.1"
-    template   = ">= 2.1"
```

Comment:
Can you please also remove the template provider from required providers in [examples](https://github.com/terraform-aws-modules/terraform-aws-eks/tree/master/examples).